### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by ThEWiZ76**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,2 +1,11 @@
+# please use comments in all your code kids!
+
 FROM alpine:3.14
-CMD ["echo", "Octopus"]
+
+MAINTAINER "Serializator"
```
Comment (Serializator) [inline Dockerfile]: Use of `MAINTAINER` is deprecated, please use a label for this as well.
https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

**Review by Serializator — CHANGES_REQUESTED** [review]
Please use a label instead of `MAINTAINER` as it is deprecated ❤️ 
https://docs.docker.com/engine/reference/builder/#maintainer-deprecated

File: Dockerfile (hunk 1 above)
Comment (ThEWiZ76) [reply Dockerfile]: updated the PR with the requested improvements

Comment (ThEWiZ76) [reply Dockerfile]: @Serializator did you have a look?

**Pull request closed without merging**